extensions/webidl/00_webidl.js

```diff
@@ -921,9 +926,7 @@
 
   const _iteratorInternal = Symbol("iterator internal");
 
-  const globalIteratorPrototype = ObjectGetPrototypeOf(ObjectGetPrototypeOf(
-    [][SymbolIterator](),
-  ));
+  const globalIteratorPrototype = ObjectGetPrototypeOf(ArrayIteratorPrototype);
```

Comment:
Not really sure if this change is fine. 
Otherwise, could `[][SymbolIterator]()` be replaced with `primordials.ArrayPrototype[SymbolIterator]()`? If it could I'd be happy to replace it.

Reply:
I'm also not entirely sure if this is correct, but I'm going to tentatively land it